koku/masu/celery/tasks.py

```diff
@@ -277,6 +279,79 @@ def clean_volume():
     LOG.info("The following files were deleted: %s", deleted_files)
 
 
+@celery_app.task(name="masu.celery.tasks.get_daily_currency_rates", queue=DEFAULT)
+def get_daily_currency_rates():
+
+    # conversionRates = {}
+    # List of the 15 supported currencies
+    # currencyList = [
+    #     "USD",
+    #     "AUD",
+    #     "CAD",
+    #     "CHF",
+    #     "CNY",
+    #     "DKK",
+    #     "EUR",
+    #     "GBP",
+    #     "HKD",
+    #     "JPY",
+    #     "NOK",
+    #     "NZD",
+    #     "SEK",
+    #     "SGD",
+    #     "ZAR",
+    # ]
+    endpoint = "http://api.exchangeratesapi.io/v1/latest?access_key=76b80adfa0400a63c7a8c8e5bc843b89"
```

Comment:
Should we store this access key as a secret?

Reply:
This has been done, now imports as     endpoint = settings.CURRENCY_ENDPOINT